### PR TITLE
Refactor/clean up routes

### PR DESCRIPTION
**Pull request by nvdk**
This refactors the routes a bit moving them into separate files split on the provided function (preview, signing, publication). which should give us a better overview. I could also have split on the type of publication (agenda, decision list, etc.) but this seems to make more sense atm.

depends on #26

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,28 +1,29 @@
...
+To facilitate this it also provides calls to generate a preview of the document as it will be published/signed.
 
-- `/prepublish/agenda/:zittingIdentifier`: builds a publication from the agenda of a zitting present in the database, and returns it without persisting it
-- `/signing/agenda/sign/:kind/:zittingIdentifier`:  persists (if it doesn't exist yet) an agenda publication for :kind and creates a signedResource linked to the agenda.
-- `/signing/agenda/publish/:kind/:zittingIdentifier`:  persists  (if it doesn't exist yet) an agenda publication for :kind and creates a publishedResource linked to the agenda.
+Once a specific document is generated for signing or publication the content is "locked". The prepublish service will always reuse the previously generated document for new signatures or publications. This means that if for example this service is used to sign a decision list, the same generated document will be used for publication even when changes have been made to the linked decisions in the mean time. An exception is the notulen publication, as the notulen publication can remove content it will: TODO verify how it's currently set up.
```
Comment (nvdk) [inline README.md:5]: should update the TODO after we've verified (and document) how the notulen publication works

File: app.js [hunk 2]
```diff
@@ -1,335 +1,10 @@
...
-    // @ts-ignore
-    error.status = 500;
-    return next(error);
-  }
-});
+import previewRouter from './routes/preview';
+import signingRouter from './routes/signing';
+import publicationRouter from './routes/publication';
 
+app.use(previewRouter);
```
Comment (nvdk) [inline app.js:7]: probably best if this approach also works in a newer javascript template

File: routes/publication.js [hunk 3]
```diff
@@ -0,0 +1,122 @@
+import express from 'express';
+import {getZittingForBesluitenlijst} from '../support/besluit-queries';
```
Comment (Asergey91) [inline routes/publication.js:2]: this seems to be missing an import 
`import {getZittingForAgenda} from "../support/agenda-queries";`

**Review by Asergey91 — CHANGES_REQUESTED** [review]
I have tested with mu-js template 1.5.0 beta 2 and everything seems to work, might as well bump the js template.

**Comment by nvdk** [comment]
rebased branch onto master

**Comment by abeforgit** [comment]
this doesn't build for me
```
SyntaxError: /usr/src/app/app/node_modules/mkdirp/bin/cmd.js: 'return' outside of function. (13:4)

  11 | if (argv.help) {
  12 |     fs.createReadStream(__dirname + '/usage.txt').pipe(process.stdout);
> 13 |     return;
     |     ^
  14 | }
  15 |
  16 | var paths = argv._.slice();
    at Parser._raise (/usr/src/app/node_modules/@babel/parser/lib/index.js:807:17)
    at Parser.raiseWithData (/usr/src/app/node_modules/@babel/parser/lib/index.js:800:17)
    at Parser.raise (/usr/src/app/node_modules/@babel/parser/lib/index.js:761:17)
    at Parser.parseReturnStatement (/usr/src/app/node_modules/@babel/parser/lib/index.js:12835:12)
    at Parser.parseStatementContent (/usr/src/app/node_modules/@babel/parser/lib/index.js:12513:21)
    at Parser.parseStatement (/usr/src/app/node_modules/@babel/parser/lib/index.js:12465:17)
    at Parser.parseBlockOrModuleBlockBody (/usr/src/app/node_modules/@babel/parser/lib/index.js:13054:25)
    at Parser.parseBlockBody (/usr/src/app/node_modules/@babel/parser/lib/index.js:13045:10)
    at Parser.parseBlock (/usr/src/app/node_modules/@babel/parser/lib/index.js:13029:10)
    at Parser.parseStatementContent (/usr/src/app/node_modules/@babel/parser/lib/index.js:12541:21) {
  loc: Position { line: 13, column: 4 },
  pos: 313,
  code: 'BABEL_PARSE_ERROR',
  reasonCode: 'IllegalReturn'
}
The command '/bin/sh -c /usr/src/app/build-production.sh' returned a non-zero code: 1
ERROR: Service 'preimporter' failed to build : Build failed
```

**Comment by nvdk** [comment]
> this doesn't build for me

hmm must admin I bumped the template without checking, will have a look later today.

**Comment by nvdk** [comment]
@abeforgit I've reverted to template bump and it now builds correctly for me

**Comment by abeforgit** [comment]
the force push tripped me up for a bit fyi

**Comment by abeforgit** [comment]
the force push tripped me up for a bit fyi

**Review by abeforgit — APPROVED** [review]
seems to work
appreciate the split

**Pull request merged**